Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -193,8 +193,11 @@ struct CombinatorialKalmanFilterOptions {
   /// The trivial propagator options
   PropagatorPlainOptions propagatorPlainOptions;
 
-  /// The reference Surface
-  const Surface* referenceSurface = nullptr;
+  /// The filter target surface
+  const Surface* filterTargetSurface = nullptr;
+
+  /// The smoothing target surface
+  const Surface* smoothingTargetSurface = nullptr;
```

Comment:
Maybe we could adapt the comment to explain what the variable is used for or leave it, if we think the variable name is already descriptive enough.

Reply:
I think for now it is fine but I am happy about suggestions